sashimi/gui/main_gui.py

```diff
@@ -125,20 +129,31 @@ def __init__(self, st: State, timer):
             1: "Calibration",
             2: "Planar",
             3: "Volume",
+            4: "Scanless",
         }
 
         self.wid_paused = PausedWidget()
         self.wid_calibration = CalibrationWidget(st, st.calibration, self.timer)
         self.wid_single_plane = SinglePlaneScanningWidget(st)
         self.wid_volume = VolumeScanningWidget(st, self.timer)
+        self.wid_scanless = ScanlessWidget(st, self.timer)
 
         self.addTab(self.wid_paused, self.option_dict[0])
```

Comment:
why not just not add the tabs if they are not available?

Reply:
Aesthetics: A view of shady tabs gives the feeling of an august subyacent structure with plenty, if not endless, options, that the user is eager to explore and know more about

Reply:
Much as I appreciate the argumentation, I still thing we should aim for a (misleading) impression of elegance and simplicity :)

Reply:
In this case I beg to differ: There is no way to access those fields from the GUI but they are still there as a reflection that they can be accessed somehow.